articles/java/spring-framework/getting-started-with-spring-cloud-function-in-azure.md

```diff
@@ -103,7 +104,6 @@ Create a *src/main/azure* folder and add the following Azure Functions configura
     "AzureWebJobsStorage": "",
     "FUNCTIONS_WORKER_RUNTIME": "java",
     "FUNCTIONS_EXTENSION_VERSION": "~4",
-    "MAIN_CLASS":"com.example.DemoApplication",
```

Comment:
Are you 100% sure we don't need the main class anymore? I remember I added this specifically as this was causing a bug

Reply:
Hi @jdubois , you already validated this on your side right, we don't need this appsetting with latest spring cloud dependency.

Reply:
Sorry @kaibocai I still have this on `https://github.com/Azure-Samples/hello-spring-function-azure/blob/master/src/main/azure/local.settings.json`, I'll do to do one more test

Reply:
no worries, so local.settintgs.json is only be used on local running. You can try it out on local and also on azure portal. I believe the appsetting `MAIN_CLASS` is not needed. Thanks.

Reply:
Thanks! I tested and fixed it in the sample app at https://github.com/Azure-Samples/hello-spring-function-azure/commit/3137be8cc91ce2a54ac957f957081e54ec820870